api/app/Http/Controllers/Api/Auth/AuthController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Controllers\Api\Auth;
+
+use App\Actions\Auth\RegisterAction;
+use App\Actions\Auth\RegisterRequest;
+use App\Http\Controllers\Api\ApiController;
+use App\Http\Presenters\RegisterResponseArrayPresenter;
+use App\Http\Requests\Auth\RegisterHttpRequest;
+use Illuminate\Http\Request;
+
+class AuthController extends ApiController
+{
+    private RegisterAction $registerAction;
+    private $presenter;
+
+    public function __construct(
+        RegisterAction $registerAction,
+        RegisterResponseArrayPresenter $registerResponseArrayPresenter
```

Comment:
Change to UserArrayPresenter. The `register` endpoint returns the presented user entity.